content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,99 @@
+
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
```

Comment:
Remove this line `### INTRODUCTION TO RELATIONAL DATABASE (SQL)`

This page will render something like this:

```
<h1>Introduction to relational database</h1>
<h3>Introduction to relational database (SQL)</h3>
```

This is annoying and repetitive. Just let the title be the title. Did you run hugo and look at it with your eyes?  I dont think you did.